### DISCUSSION TITLE
Articles on publishing

### DISCUSSION THREAD

**Discussion by tarleb** [singlesourcepub/community#41] [General]
I'd be very interested to hear about interesting articles that discuss single source publishing and similar topics.

Anyone has some good links to share?

**Comment by tarleb** [comment]
One article that I liked because it imagines a completely new publishing structure that deals with versioning, accompanying data, and, small scale publications but still includes reviews: https://www.mdpi.com/2304-6775/7/2/40/htm
An approach like that might (arguably) make single-source publishing even more relevant.

**Comment by castedo** [comment]
Here are three papers after the publication you link above.

1) This one is mostly tech and collaboration focused:
 https://greenelab.github.io/meta-review/
The tech is super interesting but what really worth noting is the meta-review about what happen when this tech was actually used.

2) This one is also tech focused but more about computation reproducibility:
https://doi.org/10.1186/s41073-020-00095-y

3) This one might not seem that directly related to single source publishing, but it think it provides some critical insight into where some forms of scholarly publishing are heading and with what tech:
https://elifesciences.org/articles/64910
Some of the things I think are notable:
a) this is one of the founders of eLife saying some pretty radical things about how science publication review can work
b) the article itself has hypothes.is attached to it and real people have commented
c) the article itself is generated from public JATS (https://github.com/elifesciences/elife-article-xml/raw/master/articles/elife-64910-v1.xml) and with lots of open-source infrastructure code
